File: src/main/java/com/everest/featureflag/entities/Feature.java
package com.everest.featureflag.entities;

import jakarta.persistence.*;

@Entity
@Table(name = "features")
public class Feature {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(unique = true, nullable = false)
    private String featureKey;

    private boolean defaultEnabled;

    private String description;

    // getters and setters
}

File: FeatureFlagManagement/src/Feature.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FeatureFlagManagement.Entities
{
    [Table("features")]
    public class Feature
    {
        [Key]
        [Column("feature_key")]
        public string Key { get; set; }

        [Column("default_enabled")]
        public bool DefaultEnabled { get; set; }

        [Column("description")]
        public string Description { get; set; }
    }
}